Cargo.toml

```diff
@@ -66,15 +66,16 @@ sqlx = { version = "0", features = [
 ] }
 helium-anchor-gen = { git = "https://github.com/helium/helium-anchor-gen.git" }
 helium-crypto = { version = "0.8.4", features = ["sqlx-postgres", "multisig"] }
+helium-lib = { git = "https://github.com/helium/helium-wallet-rs.git", branch = "master" }
```

Comment:
time to extract `helium-lib` out of wallet? maybe push into `helium-program-library`?

Reply:
probably not? The whole point is that compiling all of HPL is very brittle and complicated.. helium-lib only relies on the IDLs for the HPL defined programs